### ISSUE TITLE
[Bugs Everywhere] 

### ISSUE THREAD

**Issue by illtellyoulater** [coqui-ai/TTS#3569]
### Describe the bug

After a few days using this project I've come to think it's the worst code base I've ever seen.

It's a massive pile of half working code full of frustrating bugs. I've even seen seriously bad stuff like code trying to run stuff in the root dir and other unacceptable stuff like that which has gone unaddressed for years.

Or fatal bugs like the one reported here, where the installation of Barks fails silently and will never be attempted again just because of a download failing on first attempt... But the worst thing is that instead of putting an alert for serious issue like this in the README, that page is only modified to show how modern and cool and up-to-date this toolkit is being maintained....while under the hood entire layers of this project are falling apart.

The only good part seem to be the frog concept, but at this point, and with all the above said, all it's really starting to look more like some facade operation... I really hope someone didn't fall for that and pour some money into this...

### To Reproduce

clone this repo

### Environment

```shell
correctly set up.
```

**Comment by jp-x-g** [comment]
Very astute and helpful observation -- it's almost as though the [company shut down a month and a half ago and stopped maintaining the repository](https://news.ycombinator.com/item?id=38856095) or something.

**Comment by illtellyoulater** [comment]
@jp-x-g: 
Shutting down a company doesn't automatically mean they stop maintaining the code. They could still choose to keep it up as an open-source project. This was neither directly supported nor denied in the HN comment you're referring to. So, keep your offensive considerations for yourself, or just STFU already.

@ Coqui Team:
After seeing the impressive performance of XTTS, and realizing it was indeed Coqui's own project (this was not immediately clear, given it was mixed in with all other libraries, especially for someone new to this corner of AI), I feel it's only right to tone down my earlier remarks. Had I been fully aware of the situation my comments would have certainly been different. 
For the technology you developed, you only deserve respect, support, and appreciation. 
Wishing you clear skies ahead, whatever new horizons you'll love to explore.

**Comment by jp-x-g** [comment]
The company did release the codebase (that's why this repository exists and is public), but per [this](https://github.com/coqui-ai/TTS/issues/3488) comment from the maintainer a couple days ago, there are no current plans to support further development on this branch. 

At any rate, there is a [template](https://github.com/coqui-ai/TTS/issues/new?assignees=&labels=bug&projects=&template=bug_report.yaml&title=%5BBug%5D+) for bug reports on this repository, with six fields and clear instructions for each; you made a post calling the project a "massive pile of half working code" and "the worst code base I've ever seen" with no specific issue, no replication instructions, no configuration or environment data, no logs, and no context. 

In the future, it would probably be helpful to keep in mind that general chat threads (and especially posts made entirely to flame the devs) probably should not be tagged as bugs.

**Comment by manmay-nakhashi** [comment]
@illtellyoulater no one works for you here, this is very inappropriate, please raise a proper bug request, closing this.

**Comment by illtellyoulater** [comment]
> but per [this](https://github.com/coqui-ai/TTS/issues/3488) comment from the maintainer a couple days ago

I acknowledge that. However, I'd like to draw attention to another comment from the same timeframe:

https://github.com/coqui-ai/TTS/issues/3488#issuecomment-1924664197

This was your comment, expressing doubt and uncertainty about whether there were plans by the repository's maintainers to continue its upkeep...

Fast forward two weeks, and here you are, arrogantly dismissing my genuine concerns as a newcomer finding himself in a repository which from the README portrays as active, vibrant, and up-to-date, while in reality lacking any immediate visible indication about its abandonment, alongside significant information about most of the code being out of date and often non-functional... Yet you took your occasion to mock me as "astute" for just expressing my frustration...

As for the point that I failed to mention any actual code issues, the whole point of this thread was obviously not that of reporting specific code issues, which I have already discussed where relevant, for example in https://github.com/coqui-ai/TTS/issues/2781#issuecomment-1934262407, where I also gave my contribution to identify the current problem and proposed a solution to fix it. This was in addition to other PR I was working on locally, before realizing the actual project's condition...

In the end, while I can somewhat comprehend the rationale behind a low-key approach to project maintenance transitioning, your deriding comments lack any reasonable justification. If I really were to speculate, it might be you're in some kind of association with Coqui's services, and that might have clouded your view. Not saying that's an excuse to act like a jerk though.
